runtime/ftplugin/man.vim

```diff
@@ -6,46 +6,9 @@ if exists('b:did_ftplugin') || &filetype !=# 'man'
 endif
 let b:did_ftplugin = 1
 
-let s:pager = !exists('b:man_sect')
-
-if s:pager
+if !exists('b:man_sect')
   call man#init_pager()
 endif
 
-setlocal buftype=nofile
-setlocal noswapfile
-setlocal bufhidden=hide
-setlocal nomodified
-setlocal readonly
```

Comment:
No, because the problem is rendered man page vs. source files.  These settings were being applied to source files as well as rendered pages, which is wrong.